api/main.py

```diff
@@ -715,9 +712,16 @@ async def put_regression(regression_id: str, regression: Regression,
 @app.middleware("http")
 async def redirect_http_requests(request: Request, call_next):
     """Redirect request with version prefix when no version is provided"""
+    response = None
     path = request.scope['path']
-    prefix = path.split('/')[1]
-    if prefix not in API_VERSIONS:
+    match = re.match(r'^/(v\d+)', path)
```

Comment:
We should also consider optional minor number in the version as we might add it in the future.

Reply:
Yes but right now it's not supported and I think we agreed to only have major numbers, so I left that out explicitly.  It's true that adding support for it in this check right now shouldn't cause side-effects, in fact the current regex would already match things like `v1.2.3`.  So yeah I'll update this :)